lib/lottery/double-play.js

```diff
@@ -0,0 +1,143 @@
+const axios = require('axios');
+const DomParser = require('dom-parser');
+
+const parser = new DomParser();
+
+const options = {
+  method: 'GET',
+  url: 'https://www.powerball.com/double-play',
+};
+
+/* function getWinninNumbers() will fetch the double play winning number of the day,
+ * return a winningNumber object. The schema is look like this:
+ * WinningNumber {
+ *   type: 'double-play',
+ *   date: '2023-07-14T17:45:41.487Z',
+ *   N1: 1,
+ *   N2: 2,
+ *   N3: 3,
+ *   N4: 4,
+ *   N5: 5,
+ *   NS: 6,
+ * }
+ */
+async function getWinningNumbers() {
+  const response = await axios.request(options);
+  const dom = parser.parseFromString(response.data);
+
+  const whiteBalls = dom.getElementsByClassName('black-balls').map((el) => parseInt(el.innerHTML, 10));
+  const NS = parseInt(dom.getElementsByClassName('dp-powerball')[0].innerHTML, 10);
```

Comment:
shouldn't this be 'specialBall' according to the comemnts in PR#7 https://github.com/lufli/next-tool/pull/7#discussion_r1264267809